app/screens/settings/index.tsx

```diff
@@ -6,17 +6,16 @@ import withObservables from '@nozbe/with-observables';
 import {of as of$} from 'rxjs';
 import {switchMap} from 'rxjs/operators';
 
-import {observeConfig} from '@queries/servers/system';
+import {observeConfigValue} from '@queries/servers/system';
 import {isValidUrl} from '@utils/url';
 
 import Settings from './settings';
 
 import type {WithDatabaseArgs} from '@typings/database/database';
 
 const enhanced = withObservables([], ({database}: WithDatabaseArgs) => {
-    const config = observeConfig(database);
-    const siteName = config.pipe(switchMap((c) => of$(c?.SiteName)));
-    const showHelp = observeConfig(database).pipe(switchMap((c) => of$(c?.HelpLink ? isValidUrl(c.HelpLink) : false)));
+    const siteName = observeConfigValue(database, 'SiteName') || of$('Mattermost');
```

Comment:
We talked about the use of `Mattermost` here and the impact with white labeling. In case the value for `siteName` is undefined, you should use the server display name by using the hook in the component.

Reply:
corrected